pkg/plugins/discovery_test.go

```diff
@@ -79,9 +80,6 @@ func TestFileSpecPlugin(t *testing.T) {
 	}
 
 	for _, c := range cases {
-		if err = os.MkdirAll(path.Dir(c.path), 0755); err != nil {
```

Comment:
Even though this code isn't needed, I suspect @calavera wrote it this way so that if a new file is added to the list of 'cases' that actually has a dir then it'll automatically create the appropriate dirs.  Just future proof.  Either way is fine tho.